overlay/files-ubuntu-arm-rpi/etc/default/console-setup

```diff
@@ -0,0 +1,7 @@
+ACTIVE_CONSOLES="/dev/tty[1-6]"
```

Comment:
same file as the ubuntu files, wont copying the ubuntu files if it matches ^ubuntu* would be enough so we dont duplicate?

Reply:
we just have the one difference with the `/etc/elemental/config.yaml` for ubuntu arm rpi. Here's where I started thinking about also refactoring this but then I thought, I better start the discussion before changing everything LOL